tests/test_spark.py

```diff
@@ -430,3 +433,75 @@ def test_task_stdout():
         assert stdout_file["size"] > 0, "stdout file should have content"
     finally:
         sdk_install.uninstall(utils.SPARK_PACKAGE_NAME, service_name)
+
+
+@pytest.mark.sanity
+def test_handling_wrong_request_to_spark_dispatcher():
+    service_name = utils.FOLDERED_SPARK_SERVICE_NAME
+    utils.require_spark(service_name=service_name)
+
+    json_data = sdk_cmd.run_cli("dcos task --json")
```

Comment:
when a test suite is running, there's going to be multiple tasks. `dcos task --json` will return an array of tasks in JSON format and a spark task can be found in it by name field. After that, the parsing below should work.